### PR TITLE
Fixed strings content

### PR DESCRIPTION
**Pull request by dashapaster**
fixed name of values, witch was - 'input' and 'result' in trim.sl, append.sl, substring.sl 
fixed description in substring.sl 
fixed substring - Issues regarding boundary values
fixed string_equals - ignoreCase input missing

### CODE REVIEW

**Comment by mihaitusa** [comment]
@dashapaster  you have tests that fails... please correct them

**Comment by dashapaster** [comment]
@mihaitusa  sure, thanks !

**Comment by dashapaster** [comment]
@mihaitusa reviews please

File: content/io/cloudslang/base/strings/string_equals.sl [hunk 1]
```diff
@@ -21,9 +23,15 @@ operation:
   inputs:
     - first_string
     - second_string
+    - ignore_case:
+         default: false
```
Comment (mihaitusa) [inline content/io/cloudslang/base/strings/string_equals.sl]: @dashapaster  Put here also required: false

File: content/io/cloudslang/base/strings/substring.sl [hunk 2]
```diff
@@ -1,43 +1,42 @@
...
  #   The Apache License is available at
  #   http://www.apache.org/licenses/LICENSE-2.0
  ####################################################
  #!!
- #! @description: Substring from begin_index to end_index.
- #! @input string: original string - Example: "good morning"
- #! @input begin_index - begin index for the substring - Example: 0 (the first index = 0)
- #! @output end_index: end index for the substring - Example: 4 (new string will not include end_index)
- #! @output result: new string - Example: "good"
+ #! @description: Substring sring from beginIndex to endIndex.
```
Comment (mihaitusa) [inline content/io/cloudslang/base/strings/substring.sl]: typo + convert inputs into snake_case to be consistent with the rest of the content

File: content/io/cloudslang/base/strings/substring.sl [hunk 3]
```diff
@@ -1,43 +1,42 @@
...
         error_message = ""
-        maxIndex=len(string)+1
-        if end_index==0:
-            result=string[begin_index:]
-        elif begin_index <= maxIndex and end_index <= maxIndex:
-           result=string[begin_index:end_index]
+        maxIndex=len(origin_string)+1
+        if endIndex==0:
+            new_string=origin_string[beginIndex:]
+        elif beginIndex <= maxIndex and endIndex <= maxIndex:
```
Comment (mihaitusa) [inline content/io/cloudslang/base/strings/substring.sl]: @dashapaster  What if beginIndex > endIndex but both are < than maxIndex ?

**Comment by mihaitusa** [comment]
@dashapaster Please handle comments.

File: content/io/cloudslang/base/strings/substring.sl (hunk 3 above)
Comment (dashapaster) [inline content/io/cloudslang/base/strings/substring.sl]: @mihaitusa  will be FAILURE, is it ok?

File: content/io/cloudslang/base/strings/string_equals.sl (hunk 1 above)
Comment (dashapaster) [inline content/io/cloudslang/base/strings/string_equals.sl]: done

File: content/io/cloudslang/base/strings/substring.sl (hunk 2 above)
Comment (dashapaster) [inline content/io/cloudslang/base/strings/substring.sl]: done

File: content/io/cloudslang/base/strings/substring.sl (hunk 3 above)
Comment (dashapaster) [inline content/io/cloudslang/base/strings/substring.sl]: @mihaitusa will be falure. i add this to error_message

**Comment by dashapaster** [comment]
@mihaitusa I fixed all , can you reveiw pls.

File: content/io/cloudslang/base/strings/substring.sl [hunk 4]
```diff
@@ -1,43 +1,42 @@
...
  #!!
- #! @description: Substring from begin_index to end_index.
- #! @input string: original string - Example: "good morning"
- #! @input begin_index - begin index for the substring - Example: 0 (the first index = 0)
- #! @output end_index: end index for the substring - Example: 4 (new string will not include end_index)
- #! @output result: new string - Example: "good"
+ #! @description: Substring sring from beginIndex to endIndex.
+ #! @input origin_string: origin_string - Example: "good morning"
+ #! @input begin_index - position in string from which we want to cut - Example: 0 (the first index = 0)
+ #! @output end_index: position in string to which we want to cut - Example: 4 (new string will not include endIndex)
```
Comment (mihaitusa) [inline content/io/cloudslang/base/strings/substring.sl]: @dashapaster ... (new string will not include endIndex) replace it with end_index

**Comment by mihaitusa** [comment]
@dashapaster  Please handle one comment (a typo)...

**Comment by dashapaster** [comment]
@mihaitusa done
